scipy/stats/stats.py

```diff
@@ -366,6 +366,9 @@ def gmean(a, axis=0, dtype=None):
         dtype of a, unless a has an integer dtype with a precision less than
         that of the default platform integer. In that case, the default
         platform integer is used.
+    weights : array_like, optional
+        The weights for each value in `a`. Default is None, which gives each
```

Comment:
Maybe add the information about dimensions from `np.average` documentation:
> The `weights` array can either be 1-D (in which case its length must be the size of `a` along the given `axis`) or of the same shape as `a`.

Reply:
Good advice. Thanks! Resolved in 05833658fc4dd28e3fa2f5d5998905b6dbdaaf93.